_transport.py

```diff
@@ -79,7 +88,13 @@ def HTTPGet(self, url, headers=None, params=None):
       return None
 
     self.LogResponseData(response)
-
+    if headers is not None:
```

Comment:
Could be shortened to:
if headers is not None and 'Authorization' in headers:
  your_function_name(arg1, arg2, ...)